modules/scripts/startup-script/README.md

```diff
@@ -171,13 +171,8 @@ they are able to do so by using the `gcs_bucket_path` as shown in the below exam
   source: ./modules/scripts/startup-script
   settings:
     gcs_bucket_path: gs://user-test-bucket/folder1/folder2
-    runners:
-      - type: shell
-        source: modules/startup-script/examples/install_ansible.sh
-        destination: install_ansible.sh
-      - type: shell
-        source: modules/startup-script/examples/install_cloud_ops_agent.sh
-        destination: install_cloud_ops_agent.sh
+    install_ansible: true
```

Comment:
nit: I would prefer not to put the `install_ansible` in the example. Most common use case is that users will not define this setting and use the default behavior.